packages/grid/src/features/standard-table/components/TextCell.tsx

```diff
@@ -1,17 +1,21 @@
-import { Indent, Text } from "@stenajs-webui/core";
+import { Indent, Text, TextSize } from "@stenajs-webui/core";
 import * as React from "react";
 import styles from "./TextCell.module.css";
 
 interface Props {
   label?: string;
+  size?: TextSize;
+  color?: string;
 }
 
 export const TextCell: React.FC<Props> = React.memo(function TextCell({
   label,
+  size,
+  color,
 }) {
   return (
     <Indent overflow={"hidden"}>
-      <Text className={styles.textCell} title={label}>
+      <Text className={styles.textCell} title={label} size={size} color={color}>
```

Comment:
I believe that a similar thing is needed for `renderCell`/`createStandardEditableTextCell` as the default text size is not applied there

![image](https://github.com/StenaIT/stenajs-webui/assets/37072903/6d9517f1-32c7-4462-bb5a-3abc9c506888)

Reply:
Or wait. It is always supposed to be custom content on how to display the editable cell?

Reply:
renderCell already has full control over how to render the cell.
But yes, createStandardEditableTextCell should allow textSize as well.

I have added a commit.